packages/now-go/go-helpers.ts

```diff
@@ -156,5 +166,20 @@ export async function downloadGo(
         .on('finish', resolve);
     });
   }
-  return createGo(dir, platform, arch);
+  return createGo(workPath, dir, platform, arch);
+}
+
+async function parseGoVersion(modulePath: string): Promise<string> {
+  let version = '1.16';
+  const file = join(modulePath, 'go.mod');
+  if (!existsSync(file)) {
```

Comment:
I'd prefer to `try/catch` the `readFile()` function and print the `debug()` line when `err.code === 'ENOENT'`.